src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -151,6 +153,44 @@ const WorkspaceAccessError = () => {
   ])
 }
 
+const checkBucketAccess = withErrorReporting('Error checking bucket access', async (signal, namespace, name) => {
+  try {
+    await Ajax(signal).Workspaces.workspace(namespace, name).checkBucketReadAccess()
+    return true
+  } catch (error) {
+    if (error.status === 403) {
+      notify('error', div([
+        'The Google Bucket associated with this workspace is currently unavailable. This should be resolved shortly. If this persists for more than an hour, please ',
+        link({
+          onClick: () => contactUsActive.set(true),
+          style: { color: 'white' },
+          hover: {
+            color: 'white',
+            textDecoration: 'underline'
+          }
+        }, ['contact us', icon('pop-out', { size: 10, hover: { textDecoration: 'underline' } })]),
+        ' for assistance.'
+      ]))
+    } else if (error.status === 404) {
+      notify('error', div([
+        'The Google Bucket associated with this workspace does not exist. Please ',
```

Comment:
TOL: The API docs suggest that a 404 here indicates that the workspace itself doesn't exist. Do we actually expect this to happen in normal usage, or does this fall into the category of "unexpected error", such that we could just let it fall to the default error handler?

Reply:
The default error handler handles it pretty well should we end up in a weird case where a user is trying to use a workspace that doesn't exist. 
![Screen Shot 2019-05-13 at 10 18 38 AM](https://user-images.githubusercontent.com/42386483/57628852-b37ef000-7568-11e9-8b21-03705d122b9e.png)


Including the special 404 handling her would allow users to directly open the contact us modal from the banner, but I don't think that's enough to warrant having the extra code